ckanext/harvest/model/__init__.py

```diff
@@ -242,92 +248,93 @@ def define_harvester_tables():
     global harvest_log_table
 
     harvest_source_table = Table('harvest_source', metadata,
-        Column('id', types.UnicodeText, primary_key=True, default=make_uuid),
-        Column('url', types.UnicodeText, nullable=False),
-        Column('title', types.UnicodeText, default=u''),
-        Column('description', types.UnicodeText, default=u''),
-        Column('config', types.UnicodeText, default=u''),
-        Column('created', types.DateTime, default=datetime.datetime.utcnow),
-        Column('type',types.UnicodeText,nullable=False),
-        Column('active',types.Boolean,default=True),
-        Column('user_id', types.UnicodeText, default=u''),
-        Column('publisher_id', types.UnicodeText, default=u''),
-        Column('frequency', types.UnicodeText, default=u'MANUAL'),
-        Column('next_run', types.DateTime),
-    )
+                                 Column('id', types.UnicodeText, primary_key=True, default=make_uuid),
```

Comment:
I'd be tempted to put a line break in after `Table(` and keep everything to the left side, minimizing wrapping, which suits editing on small screens, or with side-by-side windows.